### PR TITLE
WIP - Watched Lots

### PR DESCRIPTION
**Pull request by xtina-starr**
The type of this PR is: **TYPE**

This PR resolves [CX-]

### PR Checklist (tick all before merging)

- [ ] I have included screenshots or videos to illustrate my changes, or I have not changed anything that impacts the UI.
- [ ] I have added tests for my changes, or my changes don't require testing, or I have included a link to a separate Jira ticket covering the tests.
- [ ] I have documented any follow-up work that this PR will require, or it does not require any.
- [ ] I have added an app state migration, or my changes do not require one. ([What are migrations?](https://github.com/artsy/eigen/blob/master/docs/adding_state_migrations.md))
- [ ] I have added a [CHANGELOG.yml](/CHANGELOG.yml) entry or my changes do not require one.

[CX-434]: https://artsyproduct.atlassian.net/browse/CX-434

### CODE REVIEW

File: src/lib/Scenes/MyBids/Components/WatchedLot.tsx [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+// import { TimelySale } from "../helpers/timely"
+// import { HighestBid, Outbid, ReserveNotMet } from "./BiddingStatuses"
+import { LotFragmentContainer as Lot } from "./Lot"
+
+interface WatchedLotProps {
+  lotStanding: WatchedLot_lotStanding
+}
+
+export const WatchedLot: React.FC<WatchedLotProps> = ({ lotStanding }) => {
+  console.log("lotStandingggggg:", JSON.stringify(lotStanding, null, 4))
```
Comment (erikdstock) [inline src/lib/Scenes/MyBids/Components/WatchedLot.tsx:20]: the lot standing prop here should probably just be renamed `lot`.

The goal here is to have a new "lot" type which will add causality lot state to gravity sale artwork, and then for "lot standing" to be the lot type (in a duck typing sense) plus bidder-specific information. Lot/Lot Standing thus reflect real-world vocabulary and also unite our interfaces in a way we can use `WatchedLot` (with a `lot`) with the same interface as `ActiveLot`/`ClosedLot` (which require a `lotStanding`).

sale artwork = a lot in an auction, but an older interface derived only from gravity
lot = a lot in an auction `lot` (state from causality) plus a `saleArtwork` (gravity data about the sale/artwork)
lot standing = a lot plus the user's current status on it (bid amount, whether they are winning etc)

File: src/lib/Scenes/MyBids/MyBids.tsx [hunk 2]
```diff
@@ -146,6 +153,9 @@ const MyBids: React.FC<MyBidsProps> = (props) => {
             </Join>
           </Flex>
         )}
+        {watchedLots.map((ls) => {
```
Comment (erikdstock) [inline src/lib/Scenes/MyBids/MyBids.tsx:156]: Related to my comment about duck typing above, according to the spec lots should be displayed in order by lot number (after grouping by sale) without regard for whether a bidder has a lot standing (active/closed lot component) or is only watching. There may be a better way to do this than the transformations I have up above but we ultimately want to have a list like:

### Artsy prints and multiples
------------
Lot 1: Winning
Lot 6: Watching
Lot 7: Outbid
Lot 99: Watching

**Review by erikdstock — COMMENTED** [review]
Realize this is a draft PR but as we may be blocked on the metaphysics side I wanted to get some early feedback in.

File: src/lib/Scenes/MyBids/Components/WatchedLot.tsx (hunk 1 above)
Comment (xtina-starr) [reply src/lib/Scenes/MyBids/Components/WatchedLot.tsx:20]: Yeah this makes sense.

File: src/lib/Scenes/MyBids/MyBids.tsx (hunk 2 above)
Comment (xtina-starr) [reply src/lib/Scenes/MyBids/MyBids.tsx:156]: Ahh that adds a bit more complexity. So the partition basically is determined by whether the lot is closed or not (what controls `isClosed`)?  

Could we do something like create an array that merges `lotStandings` and `watchedLots` then remove duplicate  `watchedLots` (i.e. watchedLots that already represented by a bid/lotStanding) and do the partition from there?

File: src/lib/Scenes/MyBids/Components/WatchedLot.tsx (hunk 1 above)
Comment (ashleyjelks) [reply src/lib/Scenes/MyBids/Components/WatchedLot.tsx:20]: @erikdstock to clarify...`sale artwork`, `lot`, and `lot standing` are the same regardless of if its a timed or live auction?

Comment (MounirDhahri) [reply src/lib/Scenes/MyBids/Components/WatchedLot.tsx]: I finally understood the difference between (saleArtwork, lot and lotStanding). Thank you @erikdstock ✨✨✨ 🙇‍♂️

**Pull request closed without merging**